app/controllers/admin/videos_controller.rb

```diff
@@ -1,12 +1,12 @@
 class Admin::VideosController < Admin::BaseController
-  def edit
-    @video = Video.find(params[:video_id])
-  end
+  # def edit
+  #   @video = Video.find(params[:video_id])
+  # end
 
-  def update
-    video = Video.find(params[:id])
-    video.update(video_params)
-  end
+  # def update
+  #   video = Video.find(params[:id])
+  #   video.update(video_params)
+  # end
```

Comment:
There are no routes for update/edit videos. If we were continuing to work on this project we should build them out, but right now they are hurting our coverage and unused

Reply:
I'm on board!